Add out-of-mesh server metadata info into telemetry v2 faq. (#9017)
istio/istio.io: content/en/faq/metrics-and-logs/telemetry-v1-vs-v2.md

  missing peer attributes like workload name, namespace and labels.
  However, if both peers have proxies injected all the labels mentioned
  [here](/docs/reference/config/metrics/) are available in the generated metrics.

>   At Istio 1.8 release, a solution was added to distribute server workload metadata

Reviewer: ```suggestion
Starting with the Istio 1.8 release, a solution was added to distribute server workload metadata
```

>   to client sidecar, which makes client side metric have server workload metadata labels

Reviewer: ```suggestion
  to the client sidecar causing client side metrics to have server workload metadata labels
```

Reviewer 2: I suggest removing "Starting with the Istio 1.8 release, a solution was added" as I don't believe the history is important...especially in future releases. 

When the server workload is out of the mesh, server workload metadata is distributed to client sidecar, causing client side metrics to have server workload metadata labels filled.

Author: Updated to remove 1.8 release. thanks!

>   filled even when the server workload is out of the mesh.

Reviewer 2: The way the section reads now is a bit confusing. If starting from 1.8, it now has workload labels for server workloads that are not part of the mesh, then why not just remove this whole section? It seems strange to say there is a problem and then say the solution is there since an older Istio release.

Author: Stats reporting both at client proxy and server proxy. The 1.8 solution only resolves missing server metadata at client side. Stats reporting at server side still lacks metadata is the client is outside the mesh.
